Apps/WebClient/src/ClientApp/src/components/modal/rating.vue

```diff
@@ -0,0 +1,105 @@
+<script lang="ts">
+import Vue from "vue";
+import { Component, Prop } from "vue-property-decorator";
+import { ILogger } from "@/services/interfaces";
+import { SERVICE_IDENTIFIER } from "@/plugins/inversify";
+import container from "@/plugins/inversify.config";
+
+@Component
+export default class RatingComponent extends Vue {
+    private logger: ILogger = container.get(SERVICE_IDENTIFIER.Logger);
+    @Prop({
+        default:
+            "Did the Health Gateway improve your access to health information today? Please provide a rating.",
+        required: false,
+    })
+    question!: string;
+    private ratingValue: number = 0;
+    private isVisible: boolean = false;
+    public showModal() {
+        this.isVisible = true;
+    }
+    public hideModal() {
+        this.isVisible = false;
+    }
+    private handleRating(value: number, skip: boolean = false) {
```

Comment:
handleRating takes both value and if it was skipped. With that there is no need for the skip method at all. The button can call this function with different parameters